lib/themes/dosomething/paraneue_dosomething/js/components/carousel.js

```diff
@@ -0,0 +1,30 @@
+const React = require('react');
+
+import ContextSlide from './context_slide.js';
+
+const Carousel = React.createClass({
```

Comment:
`React.createClass` was the original method for component creation. Not officially deprecated yet, but Facebook are [moving away from using it internally](https://github.com/reactjs/core-notes/blob/9d503233bdfacb09c28ac29f4da227873e40067d/2016-06/june-30.md#update-on-createclass--es-classes) and it's reasonable to assume it might be removed at some point in the future.